changelog.md

```diff
@@ -1,5 +1,111 @@
+  * [macOS] Use `-stdlib=libc++` ([#1361](https://github.com/SFML/SFML/pull/1361))
+  * [macOS] Fix CMake warning since CMake 3.9 ([#1371](https://github.com/SFML/SFML/pull/1371))
+  * [iOS] Use a CMake toolchain file for iOS build ([#1269](https://github.com/SFML/SFML/pull/1269))
+  * [iOS] Install extlibs if needed ([#1348](https://github.com/SFML/SFML/pull/1348))
+  * [iOS] Drop 32 bit support ([#1374](https://github.com/SFML/SFML/pull/1374))
+  * [iOS] Force correct iOS architecture for cmake ([#1377](https://github.com/SFML/SFML/pull/1377))
+  * [iOS] Find OpenAL correctly ([#1376](https://github.com/SFML/SFML/pull/1376))
+  * [Android] Increased minimum API version to 14 ([#1362](https://github.com/SFML/SFML/pull/1362))
+  * [Android] Removed custom toolchain and added support for the newest NDK version ([#1350](https://github.com/SFML/SFML/pull/1350))
+
+### System
+
+**Bugfixes**
+
+  * Fixed crash when `sf::Clock` is constructed in a global scope ([#1258](https://github.com/SFML/SFML/pull/1258))
+
+### Window
+
+**Features**
+
+  * Implemented Cursor API ([#827](https://github.com/SFML/SFML/pull/827))
+  * Implemented Clipboard API ([#1221](https://github.com/SFML/SFML/pull/1221))
+  * [Windows] Added support for interfacing with joysticks via DirectInput when it is available ([#1326](https://github.com/SFML/SFML/pull/1326))
+
+**Bugfixes**
+
+  * Updated documentation ([#1255](https://github.com/SFML/SFML/pull/1255), [#1310](https://github.com/SFML/SFML/pull/1310))
+  * [macOS] Fixed window menu not working ([#1193](https://github.com/SFML/SFML/pull/1193))
+  * [macOS] Removed unused variable ([#1276](https://github.com/SFML/SFML/pull/1276))
+  * [macOS] Fixed crash with application messing hardware detection e.g. TeamViewer ([#1323](https://github.com/SFML/SFML/pull/1323))
+  * [macOS] Added support for (some) Hat/POV axis ([#1248](https://github.com/SFML/SFML/pull/1248))
+  * [Linux] Fixed `sf::Window::getPosition()` returning incorrect position because of differences in window managers ([#1228](https://github.com/SFML/SFML/pull/1228), [#1266](https://github.com/SFML/SFML/pull/1266))
+  * [Linux] Fix X11 key repeat handling not filtering out events from other windows ([#1291](https://github.com/SFML/SFML/pull/1291), [#1223](https://github.com/SFML/SFML/pull/1223), [#1230](https://github.com/SFML/SFML/pull/1230))
+  * [Linux] Restore fullscreen of a non-visible window ([#1339](https://github.com/SFML/SFML/pull/1339))
+  * [iOS] Fix build error when using @import ([#1272](https://github.com/SFML/SFML/pull/1272))
+  * [iOS] Don't need to find vorbisfile or vorbisenc ([#1347](https://github.com/SFML/SFML/pull/1347))
+  * [Windows] Prevent uninitialized read by zeroing memory ([#1264](https://github.com/SFML/SFML/pull/1264))
+  * [Windows] Fixed modifier keys handling ([#1357](https://github.com/SFML/SFML/pull/1357))
+  * [Android] Return correct key code for delete/backspace ([#1362](https://github.com/SFML/SFML/pull/1362))
```

Comment:
OS labels used to be ordered alphabetically (Android, iOS, Linux, macOS, Windows).